.circleci/config.yml

```diff
@@ -79,13 +79,13 @@ workflows:
   server:
     jobs:
       - lint-server
-      - deploy-server:
-          context:
-            - tpc-staging
-          filters:
-            branches:
-              only:
-                - staging
+      # - deploy-server:
```

Comment:
Disable deployment to staging server. Staging server is broken rn because of billing issues